packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -13,6 +14,12 @@ type NestedKeyOf<Obj extends object> = {
   [K in keyof Obj]: Obj[K] extends object ? NestedKeyOf<Obj[K]> : K;
 }[keyof Obj];
 
+export const ethereumNetworks = ['mainnet', 'goerli', 'kovan'];
+export const gnosisNetworks = ['gnosis', 'sokol', 'xdai'];
```

Comment:
Gnosis is the same as xdai (xdai renamed to gnosis). I would remove it from the array so that we don't confuse anyone thinking these are two separate networks. Maybe a better approach is to rename the name from "xdai" to "gnosis" before it enters any mapping/filtering functions?

Reply:
The main dependency today is the wallet, so I'll migrate the wallet to reference only gnosis first, then I'll circle back to this file and remove all xdai reference